### PR TITLE
DDF-04448 Gazetteer Home button cleanup

### PR DESCRIPTION
**Pull request by mojogitoverhere**
#### What does this PR do?
- Updates the Home button to look more like an actual button paired with its dropdown.
- Other miscellaneous CSS fixes 
#### Who is reviewing it? 

@BernardIgiri 
@willwill96 
@gjvera 
@djblue 
@rymach 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@bdeining

#### How should this be tested?

Verify toolbar functionality works as expected and looks good with different zoom levels, screen sizes, spacing, and themes
#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4448 

#### Screenshots

Old
screen shot 2019-02-26 at 7 36 17 am

New
screen shot 2019-02-26 at 7 36 00 am

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.less [hunk 1]
```diff
@@ -76,13 +75,25 @@
     .is-split-button button,
     .is-button {
       height: @minimumButtonSize;
-      line-height: @minimumButtonSize;
       display: inline-block;
-      margin-left: @mediumSpacing;
+      text-align: middle;
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.less]: ❓ Should this be center? I don't think middle is a valid value (but maybe browsers correct it for you)

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/index.tsx [hunk 2]
```diff
@@ -31,55 +31,40 @@ type State = {
...
-  display: flex;
+  white-space: nowrap;
+  background-color: ${props => props.theme.primaryColor};
 `
 const Menu = styled.div`
   position: absolute;
   top: ${props => props.theme.minimumButtonSize};
-  padding: 0;
+  padding-top: ${props => props.theme.minimumSpacing};
+  padding-bottom: ${props => props.theme.minimumSpacing};
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/index.tsx]: ✏️ Could also right these two lines as: 
```suggestion
  padding: ${props => props.theme.minimumSpacing} 0 ${props => props.theme.minimumSpacing};
```

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/index.tsx [hunk 3]
```diff
@@ -31,55 +31,40 @@ type State = {
...
   z-index: ${props => props.theme.zIndexDropdown};
   background-color: ${props => props.theme.backgroundDropdown};
+  opacity: 1 !important;
+  border: 1px solid ${props => props.theme.backgroundModal};
 `
 
 const DefaultButton = styled.button`
-  flex: 1;
+  padding-right: ${props => props.theme.largeSpacing};
+  padding-left: ${props => props.theme.largeSpacing};
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/index.tsx]: Same sort of thing here:
```suggestion
  padding: 0 ${props => props.theme.largeSpacing} 0 ${props => props.theme.largeSpacing};
```

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/zoomToHome.tsx [hunk 4]
```diff
@@ -25,16 +25,16 @@ type Props = {
 }
 const Button = styled.button`
   flex: 1;
+  margin-left: ${props => props.theme.mediumSpacing};
+  margin-right: ${props => props.theme.mediumSpacing};
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/zoomToHome.tsx]: and here:
```suggestion
  margin-right: 0 ${props => props.theme.mediumSpacing} 0 ${props => props.theme.mediumSpacing};
```

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/index.tsx (hunk 2 above)
Comment (mojogitoverhere) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/index.tsx]: IMO the explicit attributes are more readable if only two sides are being set

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/index.tsx (hunk 3 above)
Comment (mojogitoverhere) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/index.tsx]: IMO the explicit attributes are more readable if only two sides are being set

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/zoomToHome.tsx (hunk 4 above)
Comment (mojogitoverhere) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/zoomToHome.tsx]: IMO the explicit attributes are more readable if only two sides are being set

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.less (hunk 1 above)
Comment (mojogitoverhere) [reply ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.less]: Nice catch. Removed because it wasn't having any effect

**Comment by mojogitoverhere** [comment]
build now

**Comment by mojogitoverhere** [comment]
build now

**Comment by willwill96** [comment]
Note: I noticed two issues with the home location functionality unrelated to this PR's changes (tested on master):
1. If you are zoomed far enough out, the home location is always set to a point centered on Africa. This happens on the default zoom level of DDF.
2. If you click `Set Home` and you didn't change the location (i.e clicking it a second time), there is no announcement initially, but the next time you update user preferences, you get an announcement saying the new home location was saved successfully.
🌮 Hero Successful 🌮 
Changed theme settings and didn't notice anything strange happening to the styling of the home button

**Comment by bdeining** [comment]
build now

**Comment by mojogitoverhere** [comment]
build now

**Pull request merged**